FreeBSD build fails with `--enable-qsv`

### Problem Description

Build fails with `--enable-qsv` configure option.
In this case, libhb includes libvpl headers and on FreeBSD, they are used system library that is installed by ports/pkgs.
Include patch should be `${LOCALBASE}/include/vpl` , not `${CONTRIB.build}/include/vpl`.

I will soon make a pull request for this issue.

### Activity Log, Crash Log or any other details

```text
My build log shows as follows.


  : /usr/bin/clang -std=gnu99 -pipe -fmessage-length=0 -Wall -g0 -O3 -mfpmath=ss
e -msse2 -fstack-protector-strong -D_FORTIFY_SOURCE=2 -DLIBICONV_PLUG -D__LIBHB_
_ -DUSE_PTHREAD -DSYS_FREEBSD -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64 -DHAVE_
THREADS=1 -DHAS_STRERROR_R -DARCH_X86_64 -I./libhb/ -I./contrib/include -I./cont
rib/include/vpl -I/usr/local/include -I/usr/local/include/libxml2 -c ../libhb/de
cavcodec.c -o libhb/decavcodec.o
  : In file included from ../libhb/decavcodec.c:53:
  : ./contrib/include/libavutil/hwcontext_qsv.h:22:10: fatal error: 'mfxvideo.h' file not found
  : #include <mfxvideo.h>
  :          ^~~~~~~~~~~~
```
```


### What Operating System are you running?

FreeBSD current

### What version of HandBrake are you running?

head fa7e239a0b3104a7a1d303c94b50baa38a0e4ded

### Where did you download HandBrake from?

github

--- a/libhb/module.defs
+++ b/libhb/module.defs
@@ -46,7 +46,11 @@ LIBHB.GCC.D += __LIBHB__ USE_PTHREAD
 LIBHB.GCC.I += $(LIBHB.build/) $(CONTRIB.build/)include
 
 ifeq (1,$(FEATURE.qsv))
-    LIBHB.GCC.I += $(CONTRIB.build/)include/vpl
+    ifeq ($(HOST.system),freebsd))
+        LIBHB.GCC.I += $(LOCALBASE)/include/vpl
+    else
+        LIBHB.GCC.I += $(CONTRIB.build/)include/vpl
+    endif
 endif
 
 ifneq (,$(filter $(HOST.system),freebsd netbsd))

